.github/workflows/build_and_test.yml

```diff
@@ -326,14 +326,14 @@ jobs:
 
       - name: Run cargo test
         run: |
-          ${cov_prefix} cargo test $CARGO_FLAGS $CARGO_FEATURES
+          timeout --kill-after=10m 5m ${cov_prefix} cargo test $CARGO_FLAGS $CARGO_FEATURES
 
           # Run separate tests for real S3
           export ENABLE_REAL_S3_REMOTE_STORAGE=nonempty
           export REMOTE_STORAGE_S3_BUCKET=neon-github-public-dev
           export REMOTE_STORAGE_S3_REGION=eu-central-1
           # Avoid `$CARGO_FEATURES` since there's no `testing` feature in the e2e tests now
-          ${cov_prefix} cargo test $CARGO_FLAGS --package remote_storage --test test_real_s3
+          timeout --kill-after=10m 5m ${cov_prefix} cargo test $CARGO_FLAGS --package remote_storage --test test_real_s3
```

Comment:
It seems you've got an outdated base for the PR, we also have `test_real_azure`.

Reply:
yes, I just noticed I was like few months behind :)